registry/htpasswd

```diff
@@ -0,0 +1,2 @@
+testuser:$2y$05$R44iYKyPtsSqrtNEoMO3oOSlpSxa/qIa/na3WmFuxXAF8hj4wjOyu
```

Comment:
@wanghaoran1988 , would be good to add a comment with the plaintext password so one doesn't have to decrypt the string above